samza-api/src/main/java/org/apache/samza/operators/Scheduler.java

```diff
@@ -20,22 +20,22 @@
 package org.apache.samza.operators;
 
 /**
- * Allows registering epoch-time timer callbacks from the operators.
- * See {@link org.apache.samza.operators.functions.TimerFunction} for details.
- * @param <K> type of the timer key
+ * Manages scheduling keys to be triggered later. See {@link org.apache.samza.operators.functions.ScheduledFunction}
```

Comment:
Maybe: "Allows scheduling {@link ScheduledFunction} callbacks to be invoked later".